components/core/DownloadDaemon.py

```diff
@@ -120,6 +121,11 @@ def worker(self):
                     download_id = get_id_from_gid(gid)
                     username = get_username_from_gid(gid)
                     send_status(download_id, completedLength, raw_size, username)
+                    file_name = path[-1]
+                    fpath = data['result']['files'][0]['path'].split('/')
```

Comment:
Shall we reuse the `path` variable which is already present?

Reply:
> @vivonk @agentmilindu if we are integrating minio with @VibhorCodecianGupta file management work. We should make better use of minio client SDK. It's better to fetch files from minio server rather using a separate client (minio browser) for showing up the files as it seems to be a bad practice. What are your thoughts?
> @demon-36

@kmehant  Minio is going to work both ways - it will be accessible to admin to visit Minio and view downloaded files and when user will request to download files from completed section then it will get file from Minio server and serve to user.
these changes are going to be added in next commits.